packages/e2e/test/wallet/SingleAddressWallet/delegation.test.ts

```diff
@@ -58,6 +47,16 @@ const waitForTx = async (wallet: ObservableWallet, { hash }: Transaction.TxInter
   await waitForWalletStateSettle(wallet);
 };
 
+// eslint-disable-next-line func-style, jsdoc/require-jsdoc
+function assertTxIsValid(tx: MaybeValidTx): asserts tx is ValidTx {
```

Comment:
Could export those from `wallet/test/util`, [add 'includes'](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/wallet/test/tsconfig.json#L8) in tsconfig and [import with relative path](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/wallet/test/Transaction/createTransactionInternals.test.ts#L4)

Reply:
Updated as part of the rebase+squash : [bbc0574](https://github.com/input-output-hk/cardano-js-sdk/pull/396/commits/bbc0574c21c69e324f351edefad84e317c7f46f7)